### ISSUE TITLE
update tests wiki page

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#1684]
I think it would be good to have a page with a list of test datasets (scipion testdata --list), showing what it is, number of items, and maybe the tests involded. https://github.com/I2PC/scipion/wiki/Running-Tests is not bad, but it is far from complete and show the tests themselves not the datasets. 

For example, I am developng a protocol that I would like to test using existing data (saving disc space on server!) and I need particles from at least 5 different mics: there is no other way to find it rather than comb through all dataset manually and check how many ptcls we got in each case.

**Comment by pconesa** [comment]
You want the list to discover what files are inside the datasets and avoid installing all of them?

**Comment by azazellochg** [comment]
@pconesa , no, I want a webpage showing a list of files, their type (mics/ptcls etc) and number of items. However, if nobody is gonna update it then yes, putting it into list command might be a better solution

**Comment by pconesa** [comment]
Well..with the pluginization......this will also become distributed...and the current dataset will remain there.....but plugin developers can define there own datasets and host them somewhere else....I guess something from the command line will be more functional.
